### ISSUE TITLE
TLS error: network send error: certificate verify failed

### ISSUE THREAD

**Issue by yc1037642433** [ladnir/cryptoTools#32]
I run the example of TLS channel that in frontend_cryptoTools/tutorial/Network.cpp，print error certificate verify failed and then throw an exception；
<img width="990" alt="image" src="https://user-images.githubusercontent.com/18087180/158297690-eab262bb-bcb6-4721-a97f-4e80e6f83fc2.png">

**Comment by yc1037642433** [comment]
> The tls code is no longer maintained and was experimental. Not sure what the issue is. My suggestion of to use some other existing tls socket type and then wrap that socket with channel.

Do you means the Network/TLS.cpp is no longer used?

**Comment by yc1037642433** [comment]
> The tls code is no longer maintained and was experimental. Not sure what the issue is. My suggestion of to use some other existing tls socket type and then wrap that socket with channel.

This is my test code:
<img width="786" alt="WX20220316-100441@2x" src="https://user-images.githubusercontent.com/18087180/158502314-a673d4ff-fe28-4ade-8ec3-f70733f68055.png">

**Comment by ladnir** [comment]
Correct, I wrote that tls wrapper as somewhat of an experiment and now have decided to move in a different direction. I'm actually about to release a new networking library with tlc support which replaces cryptoTools/channel. so don't really want to maintain this one.

With regards to your sample code. I don't see anything obviously wrong. You can look at the unit tests to see what it does. You should also check each error_code. Maybe one has an error that would explain the issue.

**Comment by ladnir** [comment]
Also, maybe you shouldnt use the server's key pair for the client?

**Comment by yc1037642433** [comment]
This issue maybe because of the ca file, I use another ca file that wolfssl provided, the exception disappeared

**Comment by yc1037642433** [comment]
I will use grpc to replace the network of cryptoTool
